README.md

```diff
@@ -39,6 +39,20 @@ Note that we use one registry cache (to avoid excessive downloads), but the targ
 contract that we compile. This means no interference between contracts, but very fast recompile times when making
 minor adjustments to a contract you had previously created an optimized build for.
 
+### Feature-Based Optimization
+
+You can add a [package.metadata.optimizer] section in your Cargo.toml to enable optimization for specific features. Here's an example:
+
+```toml
+[package.metadata.optimizer]
+features = ["juno", "terra"]
```

Comment:
This is cool. Can we make this a map like this (I'm not sure about the toml syntax)?

```
builds = {
    juno = { features: ["juno"] },
    terra = { features: ["terra"] },
}
```

here the idea is to have one entry per build but can put any list of features in there. This allows more complex setups like this:

```
builds = {
    juno = { features: ["cosmwasm_1_3", "tokenfactory"] },
    stargaze = { features: ["cosmwasm_1_2", "governance_gatet", "tokenfactory"] },
    terra = { features: ["cosmwasm_1_2", "special_query_xyz"] },
}
```

It ensures the amount of features in the Rust build system can stay low when the number of builds increases. It also prepares for extensibility of the build specifications.